packages/account-v2/src/containers/LoginHistoryTable/LoginHistoryTable.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { useMemo } from 'react';
+import { LoginHistory } from '@deriv/api-types';
+import { Table } from '@deriv-com/ui';
+import { formattedLoginHistoryUtils } from '../../utils/formattedLoginHistoryUtils';
+
+type TLoginHistoryProps = {
+    loginHistory: LoginHistory;
+};
+
+const headers = {
+    action: 'Action',
+    browser: 'Browser',
+    datetime: 'Date and Time',
+    ipAddress: 'IP Address',
+    status: 'Status',
```

Comment:
let's add here as const, so we will have 'header' on line 22 with correct types, rather than just string